Automattic/wp-calypso
# Fix: Make Scan_Realtime product available to UI.

### Changes proposed in this Pull Request

Fixes 1164141197617539-as-1199554500149274 (Scan section shows upsell when user has Scan Realtime)

Scan Realtime is not available to purchase as an individual product at this time, however if the product is added through the Store Admin then Calypso blue and green was not recognizing it at all and consequentially it was not showing as a purchased product on the plans/pricing page and also the Scan upsell **was** showing when going to Jetpack --> Scan.

This PR makes Scan Realtime product available to the Calypso UI for if it is added through the Store Admin or for when we may want to make it available for purchase as an individual product in the future.  It is still not available in the UI for purchase from the public, but it **is** available in the UI if it has already been purchased (through the Store Admin).
 
### Testing instructions

- Checkout and run this PR in calypso blue & green concurrently.
- Select a Jetpack site that doesn't have Scan yet
- Subscribe to Scan Realtime through the Store Admin
- Visit Plan --> Billing page (`http://calypso.localhost:3000/purchases/subscriptions/:site`) and verify that "Scan Realtime" is showing as a purchased product. (see Fig.1)
- Visit `/scan/:site` in Calypso blue & green and verify that the Scan component is showing and **not** the Scan Upsell. (See fig.2)
- Visit the `/plans/:site` in Calypso blue & green and verify that "Scan Realtime" product card is showing as a purchased product (with "Manage Subscription" button) (See Fig.3)
    - Note: There is not a correct price set for Scan Realtime because it has not been set for purchase to the public
- Verify there are no errors in the console.


**Fig.1**
<img width="1105" alt="Markup 2021-01-19 at 14 40 32" src="https://user-images.githubusercontent.com/11078128/105084984-1d9da700-5a65-11eb-84cb-2eab2b374e65.png">

**Fig.2**
<img width="1085" alt="Markup 2021-01-19 at 14 42 57" src="https://user-images.githubusercontent.com/11078128/105084999-242c1e80-5a65-11eb-82ea-a3d2c3beae08.png">

**Fig.3**
<img width="1278" alt="Markup 2021-01-19 at 14 44 15" src="https://user-images.githubusercontent.com/11078128/105085041-2ee6b380-5a65-11eb-8fc7-7296c28be838.png">
